scipy/stats/stats.py

```diff
@@ -5948,7 +5974,23 @@ def ttest_ind(a, b, axis=0, equal_var=True, nan_policy='propagate',
     ...                 random_state=np.random.default_rng(12345))
     (-1.467966985449, 0.14)
 
+    Take these two samples, one of which has an extreme tail.
+
+    >>> a = (56, 128.6, 12, 123.8, 64.34, 78, 763.3)
+    >>> b = (1.1, 2.9, 4.2)
+
+    Use the `trim` keyword to trim the input samples. Using 20% trimming,
```

Comment:
Even though we have made it pretty clear earlier, it would be good to continue repeating (in some way) the idea that this does not _just_ trim the data. So maybe "Use the `trim` keyword to perform a trimmed (Yuen) t-test." or something.

Reply:
All of the notes from this review have been addressed, so they should be ok to resolve.